app/views/notifications/_emergency_banner.html.erb

```diff
@@ -1,9 +1,9 @@
-<section id="emergency-banner-notification" class="<%= banner.campaign_class %>">
+<section id="emergency-banner-notification" class="<%= banner.campaign_class %>" role="banner">
```

Comment:
I think we could take this opportunity to remove the use of the `id` and only use classes, this makes it more consistent with [GOV.UK Publishing Components](http://govuk-component-guide.herokuapp.com/) which I'd think we probably would want to make a 'banner' component out of this work at some point. (Non-blocking comment for this though)

Related: https://csswizardry.com/2011/09/when-using-ids-can-be-a-pain-in-the-class/